pkgs/applications/networking/irc/chatterino2/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, pkgconfig, fetchFromGitHub, qt5, boost, openssl }:
```

Comment:
please use qtbase etc. as arguments instead of qt5: see https://nixos.org/nixpkgs/manual/#ssec-qt-applications for details.

Reply:
@symphorien oh, that is interesting! Thank you for the link!